jmxtrans-output/jmxtrans-output-kafka/src/test/java/com/googlecode/jmxtrans/model/output/kafka/KafkaWriterTests.java

```diff
@@ -45,42 +46,45 @@
 @RunWith(MockitoJUnitRunner.class)
 public class KafkaWriterTests {
 
-	@Mock private Producer<String, String> producer;
-	@Captor private ArgumentCaptor<KeyedMessage<String, String>> messageCaptor;
+	@Mock
+	private Producer<String, String> producer;
+	@Captor
+	private ArgumentCaptor<ProducerRecord<String, String>> messageCaptor;
 
-	@Test public void
+	@Test
+	public void
 	kafkaWriterNotNull() throws Exception {
 		assertThat(getTestKafkaWriter()).isNotNull();
 	}
 
-	@Test public void
+	@Test
+	public void
 	messagesAreSentToKafka() throws Exception {
 		KafkaWriter writer = getTestKafkaWriter();
```

Comment:
You should probably ensure that the KafkaWriter is closed properly (try-with-resources)